### PR TITLE
CC-1063: Update test to match stage description.

### PR DESCRIPTION
**Pull request by ryan-gang**

### CODE REVIEW

File: internal/test_repl_replica_getack_nonzero.go [hunk 1]
```diff
@@ -51,6 +51,12 @@ func testReplGetaAckNonZero(stageHarness *test_case_harness.TestCaseHarness) err
 	}
 	offset += GetByteOffset([]string{"REPLCONF", "GETACK", "*"})
 
+	err = master.GetAck(offset) // 37
```
Comment (rohitpaulk) [inline internal/test_repl_replica_getack_nonzero.go:54]: @ryan-gang how come the values in the comments above (line 48) and below (line 64) is also `37`?

**Review by rohitpaulk — CHANGES_REQUESTED** [review]
(comments added)

File: internal/test_repl_replica_getack_nonzero.go (hunk 1 above)
Comment (ryan-gang) [reply internal/test_repl_replica_getack_nonzero.go:54]: These are the byte offsets for the given command. Not the cumulative offset value. These were useful during writing the initial tests.
Should I remove these comments altogether ?

Comment (rohitpaulk) [reply internal/test_repl_replica_getack_nonzero.go:54]: Aha, no it's okay then

**Review by rohitpaulk — APPROVED** [review]

**Comment by qwertzguy** [comment]
This technically still doesn't match the test description as the description mentions only sending one SET command but the test sends 2 of them.

**Comment by ryan-gang** [comment]
We want 2 set of `SET` commands, it makes the tests more robust. Will update the description to mention the same.

**Pull request merged**